Problem with installing packages from github

There is an issue when I try to install package from repository 

```bash
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror ssh://git@github.com/galileo/riek.git /root/.npm/_git-remotes/git-ssh-git-github-com-galileo-riek-git-update-to-prop-types-as-an-different-package-f2c919e4: Cloning into bare repository '/root/.npm/_git-remotes/git-ssh-git-github-com-galileo-riek-git-update-to-prop-types-as-an-different-package-f2c919e4'...
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror ssh://git@github.com/galileo/riek.git /root/.npm/_git-remotes/git-ssh-git-github-com-galileo-riek-git-update-to-prop-types-as-an-different-package-f2c919e4: error: cannot run ssh: No such file or directory
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror ssh://git@github.com/galileo/riek.git /root/.npm/_git-remotes/git-ssh-git-github-com-galileo-riek-git-update-to-prop-types-as-an-different-package-f2c919e4: fatal: unable to fork
npm ERR! Linux 4.9.13-moby
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v7.7.4
npm ERR! npm  v4.1.2
npm ERR! code 128

npm ERR! Command failed: git clone --template=/root/.npm/_git-remotes/_templates --mirror ssh://git@github.com/galileo/riek.git /root/.npm/_git-remotes/git-ssh-git-github-com-galileo-riek-git-update-to-prop-types-as-an-different-package-f2c919e4
npm ERR! Cloning into bare repository '/root/.npm/_git-remotes/git-ssh-git-github-com-galileo-riek-git-update-to-prop-types-as-an-different-package-f2c919e4'...
npm ERR! error: cannot run ssh: No such file or directory
npm ERR! fatal: unable to fork
```

Current Dockerfile

``` 
FROM node:7.7-alpine

RUN mkdir -p /var/www/app

WORKDIR /var/www/app

COPY package.json /var/www/app/

RUN npm install --production
RUN npm install -g monobrow

RUN apk update \
  && apk add git

COPY . /var/www/app
```

--- a/computer/javascript/async.md
+++ b/computer/javascript/async.md
@@ -1,6 +1,5 @@
 # Async
 
-
 ### Async basics
 
 ```javascript

--- a/computer/javascript/npm.md
+++ b/computer/javascript/npm.md
@@ -17,4 +17,9 @@ Credits to @joshwnj
 In docker container you need to ensure that you have all additional elements
 installed, like git, ssh, and etc.
 
-@todo Verify what exact packages we do require to make this work more information can be found here https://github.com/galileo/knowledge/issues/1
+To be able to install packages from github repository on `alpine` you need to install two packages `git` and `openssh`
+
+```
+RUN apk update \
+  && apk add git openssh
+```

